docker-compose.yaml

```diff
@@ -19,38 +19,24 @@ services:
 
   seth-tp:
     image: sawtooth-seth:latest
-    build: ./
-    volumes:
-      - ./:/project/sawtooth-seth
-    expose:
-      - 4004
     depends_on:
       - validator
     command: seth-tp -vv --connect tcp://validator:4004
-    stop_signal: SIGKILL
 
   block-info-tp:
-    image: hyperledger/sawtooth-block-info-tp:latest
-    expose:
-      - 4004
+    image: hyperledger/sawtooth-block-info-tp:1.0.1
```

Comment:
I think we should lock this to 1.0, not the minor release.  Otherwise we have to remember to update it every time a core release happens.

Reply:
OK